### ISSUE TITLE
No way to exit from the marker/search/minimum (other than reboot)

### ISSUE THREAD

**Issue by yarda** [ttrftech/NanoVNA#124]
Reproducer:
1. marker/search/minimum

No way to return to the normal mode which allows moving of the marker (by clicking the wheel left/right). I tried marker/search/tracking option, switching the marker on/off, nothing helped. It seems the only way how to return to the "free move" mode is reboot. I guess that it's the same with the marker/search/maximum and maybe other modes as well.

**Comment by yarda** [comment]
FW 0.7.0-20200223

**Comment by DiSlord** [comment]
For return to normal leveler mode which allows moving of the marker (by clicking the wheel left/right). Ned click at top screen (or possible touch not calibrated)

**Comment by yarda** [comment]
Yup, clicking the top screen works (I have to tried multiple times, because I wasn't sure where exactly to click). For me this was really non intuitive. Could you mention this in the documentation? Thanks.
